src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/ReceiverFactoryTest.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.exceptions.ConnectionException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.ThrowableAssert.catchThrowable;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ReceiverFactoryTest {
+
+    @Test
+    public void create_should_return_client_from_client_factory() {
```

Comment:
test description doesn't look accurate, seems to be an exception case,
also do we not have junit 5 in this project? DisplayName is much nicer than `_` separating words